src/Dfe.EarlyYearsQualification.Web/Helpers/PlaceholderUpdater.cs

```diff
@@ -0,0 +1,25 @@
+namespace Dfe.EarlyYearsQualification.Web.Helpers;
+
+public class PlaceholderUpdater : IPlaceholderUpdater
+{
+    private readonly Dictionary<string, string> _placeholders = new()
+                                                                {
+                                                                    { "$[actual-year]$", DateTimeOffset.Now.Year.ToString() }
+                                                                };
+
+    public string Replace(string valueToCheck)
```

Comment:
Suggestion (minor). It's not really doing a "check". How about just naming this parameter `text`?

Reply:
Yep, wasn't sure on the name. Text is a better suggestion :-)